core/modules/user/user.install

```diff
@@ -1344,6 +1352,45 @@ function user_update_1023() {
   }
 }
 
+/**
+ * Add changed field to users table.
+ */
+function user_update_1024() {
+  // The "changed" column was renamed to "access" in system_update_136(), and
```

Comment:
`system_update_136()`?

Reply:
I don't think so. This install file contains other changes to the user table so I think it's best here, just like in Drupal 7.

Reply:
Oh I misunderstood your comment.